### DISCUSSION TITLE
有没有BFE社区即时沟通群组？

### DISCUSSION THREAD

**Discussion by arrow2012** [bfenetworks/bfe#954] [Q&A (问题和回答)] [answered]
其他的一些国产开源产品 都有社区沟通群（比如QQ群，微信群 等） 可以即时沟通，提高效率和增加社区活跃度，BFE有没有类似的即时沟通群组？

**Comment by clarinette9** [comment, marked as answer]
您好，请扫描该文档下方的二维码加入微信群：
https://github.com/bfenetworks/bfe/blob/develop/README-CN.md
